seed/static/seed/partials/data_quality_admin.html

```diff
@@ -67,7 +67,7 @@ <h3 translate>Modifying Data Quality Rules</h3>
                         </tr>
                     </thead>
                     <tbody ng-repeat="field in sortedRuleGroups()">
-                        <tr ng-repeat="rule in ruleGroups[inventory_type][field]" ng-class="{ 'warning' : rule.new, 'disabled' : !rule.enabled }" ng-click="change_rules()">
+                        <tr ng-repeat="rule in ruleGroups[inventory_type][field]" ng-class="{ 'warning' : rule.new, 'disabled' : !rule.enabled }" ng-click="rules_changed()">
```

Comment:
This was triggered on any click (not just when the values are changed). Is this OK? Should we just apply this to every sub-html element with `ng-model`?